geode-core/src/main/java/org/apache/geode/distributed/internal/InternalLocator.java

```diff
@@ -815,6 +822,10 @@ void startClusterManagementService() throws IOException {
 
     if (distributionConfig.getEnableManagementRestService()) {
       myCache.getOptionalService(HttpService.class).ifPresent(x -> {
+        ManagementService managementService = ManagementService.getManagementService(myCache);
```

Comment:
Maybe put a comment here stating why jmxManager is needed when starting management rest service

Reply:
added a comment